src/components/ImageComponent.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import CardMedia from '@material-ui/core/CardMedia';
+
+
+const useStyles = makeStyles((theme) => ({
+  imgStyle: {
+    width: '517px',
+    margin:'auto',
+    padding:'58px 0px 94px 0px',
+    [theme.breakpoints.down('xs')]: {
+      width: '250px',
+    },
+  },
+}));
+
+const ImageComponent = (props) => {
+  const classes = useStyles();
+  return (
+    <Grid item xs={12} style={{ padding: '8px 0px', margin: 'auto' }}>
```

Comment:
Can this styling be moved to makeStyles?

Reply:
Moved to makestyles.